src/main.zig

```diff
@@ -190,14 +191,38 @@ fn initLogging() void {
 		std.log.err("Couldn't create logs/latest.log: {s}", .{@errorName(err)});
 		return;
 	};
+
+    const _timestamp = std.time.timestamp();
+    var buf: [64]u8 = undefined;
+	const _timestamp_str = std.fmt.bufPrint(&buf, "{}", .{_timestamp}) catch |err| {
+        std.log.err("Couldn't format the timestamp: {s}", .{@errorName(err)});
+        return;
+    };
+
+    const _path_str = std.fmt.allocPrint(globalAllocator.allocator, "logs/ts_{s}.log", .{_timestamp_str}) catch |err| {
```

Comment:
I prefer using the `stackAllocator` if you are freeing it in the same scope. It's a lot faster, and while it may not really matter here, but it's good to get into the right habits from the start.

Also note what errors are happening here: `allocPrint` only returns OutOfMemory errors, the allocators I'm using can be assumed to never fail, so you can simply use `catch unreachable` here.